ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/platform/server/api/validation/ErrorFactoriesSpec.scala

```diff
@@ -69,6 +81,11 @@ class ErrorFactoriesSpec
             Map("category" -> "1", "definite_answer" -> "false"),
           ),
         ),
+        expectedLogEntry = ExpectedLogEntry(
+          Level.INFO,
+          "INDEX_DB_SQL_TRANSIENT_ERROR(1,cor-id-1): Processing the request failed due to a transient database error: some db transient failure",
```

Comment:
How about interpolating this string out of the `truncatedCorrelationId`, `TransientServerFailure` and `SqlTransientError`? It would make this test resilient to trivial error code changes.

Reply:
Ended up interpolating `truncatedCorrelationId`. 
Left out `TransientServerFailure.asInt` and `SqlTransientError.id` because I think it's good to have a test that checks if  an error code has such id and such category id.

Reply:
:+1: